client/flutter/lib/l10n/intl_en.arb

```diff
@@ -7,19 +7,19 @@
     "shareTheApp": "Share the App",
     "healthCheck": "Health Check",
     "feelingDistressed": "Feeling Distressed?",
-    "washHands": "Wash your hand often with soap and running water frequently",
-    "cougningAndSneezing": "When coughing and sneezing cover mouth and nose with flexed elbow or tissue",
-    "throwAwayTissue":"Throw tissue into closed bin immediately after use",
+    "washHands": "Wash your hands often with soap and running water frequently",
```

Comment:
Having both often & frequently doesn't make sense to me. Should we rewrite that?

Reply:
good point, it is there twice. I think once is enough - will change

Reply:
Oh I misunderstood originally and removed the 'washHandsFrequently' argument - is that OK or too much?

Reply:
I think he meant just the English wording.

Reply:
agreed, will put it back - don't want to change too much in 1 MR